src/UI/Checkbox.elm

```diff
@@ -97,7 +96,7 @@ renderElement renderConfig (Checkbox { message, label, state } { labelVisible })
             , Border.color Palette.primary.middle
             , Border.width 2
             , Border.rounded 8
-            , Events.onClick (message (not state))
+            , Element.onIndividualClick (message (not state))
```

Comment:
wtf, where's this function coming from? I can't seem to find it in the docs

Reply:
It's from our Internal.Element